object.js

```diff
@@ -0,0 +1,69 @@
+let favMovie = {
+  title: "How To Train Your Dragon",
+  year: 2010,
+  director: "Dean DeBlois, and Chris Sanders",
+  actors:
+    "Jay Baruchel, Gerard Butler, Jonah Hill, America Ferrara, Kit Harington, David Tennant, Cate Blanchett",
+  rating: 8.1,
+  timesWatched: 3,
+  watch() {
+    this.timesWatched += 1;
+    return `You have watched this movie ${this.timesWatched} times.`;
+  },
+  recommend(name) {
+    return `I recommend this movie to ${name} because it's a really well written movie.`;
+  },
+  setRating(newRating) {
+    return `This movie's rating has been updated to ${newRating} instead of the old rating ${this.rating}`;
```

Comment:
This function is intended to update the old rating with the new one (i.e., receiving a new rating and setting it to this.rating):
`setRating(newRating) {
  this.rating = newRating;
}
`